packages/nextjs/src/app-beta/client/clerk-react.ts

```diff
@@ -54,12 +54,12 @@ export const useEmailLink = _useEmailLink;
  * @deprecated Use imports from `@clerk/nextjs` instead.
  * For more details, consult the middleware documentation: https://clerk.com/docs/nextjs/middleware
  */
-export const useOrganization = _useOrganization;
+export const useOrganization: typeof _useOrganization = _useOrganization;
```

Comment:
This is some weird TS issue where it couldn't clearly infer the type 🤷

Reply:
This will be dropped in the next couple of weeks, so don't bother looking more into this.